g3w-admin/templates/base.html

```diff
@@ -273,7 +273,15 @@ <h4><i class="icon fa fa-check"></i> Ok!</h4>
     <!-- Bootstrap-datepicker -->
     <script src="{% static "bower_components/bootstrap-datepicker/dist/js/bootstrap-datepicker.min.js" %}"></script>
     <!-- Bootstrap-datepicker-locals -->
-    <script src="{% static "bower_components/bootstrap-datepicker/dist/locales/bootstrap-datepicker.it.min.js" %}"></script>
+    {% for l in SETTINGS.LANGUAGES %}
+    {% if l.0 != 'en' %}
+        {% with "bower_components/bootstrap-datepicker/dist/locales/bootstrap-datepicker."|add:l.0|add:".it.min.j" as datapicker_lang %}
+    <script src="{% static datapicker_lang %}"></script>
+        {% endwith %}
+    {% endif  %}
+    {% endfor %}
+    <script src="{% static "bower_components/bootstrap-datepicker/dist/locales/bootstrap-datepicker.ro.min.js" %}"></script>
+    <script src="{% static "bower_components/bootstrap-datepicker/dist/locales/bootstrap-datepicker.fr.min.js" %}"></script>
```

Comment:
Have you already tried to update [`bootstrap-datepicker`](https://github.com/uxsolutions/bootstrap-datepicker/tree/v1.9.0) to **v1.9.0**?


```suggestion
    {% for l in SETTINGS.LANGUAGES %}
        {% with "bower_components/bootstrap-datepicker/dist/locales/bootstrap-datepicker."|add:l.0|add:".min.js" as datapicker_lang %}
    <script src="{% static datapicker_lang %}"></script>
        {% endwith %}
    {% endfor %}
```

Reply:
I just try the new 1.9.0 version but the problem persists,  in fact if, for example, you try with the Romanian language with a previously saved message, see for example geese from django the date in Romanian arrives like this for example 04.24.2023 if you try to change them by means of the datepicker widget it puts 04/24/2023, because inside its local file 'bootstrap-datepicker.ro.js' it has the date formatted like this `format: "dd/mm/yyyy"`:

```js
/**
 * Romanian translation for bootstrap-datepicker
 * Cristian Vasile <cristi.mie@gmail.com>
 */
;(function($){
	$.fn.datepicker.dates['ro'] = {
		days: ["Duminică", "Luni", "Marţi", "Miercuri", "Joi", "Vineri", "Sâmbătă"],
		daysShort: ["Dum", "Lun", "Mar", "Mie", "Joi", "Vin", "Sâm"],
		daysMin: ["Du", "Lu", "Ma", "Mi", "Jo", "Vi", "Sâ"],
		months: ["Ianuarie", "Februarie", "Martie", "Aprilie", "Mai", "Iunie", "Iulie", "August", "Septembrie", "Octombrie", "Noiembrie", "Decembrie"],
		monthsShort: ["Ian", "Feb", "Mar", "Apr", "Mai", "Iun", "Iul", "Aug", "Sep", "Oct", "Nov", "Dec"],
		today: "Astăzi",
		clear: "Șterge",
		weekStart: 1,
		format: "dd/mm/yyyy"
	};
}(jQuery));
```

so you need to change the datepicker settings like I did in previous commits:

```js
// Fix dataformat form django i18n
$.fn.datepicker.dates['en']['format'] = 'yyyy-mm-dd';
$.fn.datepicker.dates['ro']['format'] = 'dd.mm.yyyy';
 ```

Reply:
@wlorenzetti Do you think it can be easily integrated with the current testing structure?

- https://playwright.dev/python/docs/intro
- https://playwright.dev/python/docs/library

I feel like I'm missing what I'm doing with all those manual page refresh..

Reply:
@Raruto it would be nice use playwright for testing, but at the moment is better try to finish the PR for v3.6 G3W-SUITE release.

As far as I have read, the natural plugin for Playtight is for pytest, but basically django and in particular G3W-SUITE use the classic unittest, I have never used PyTest.
There's a module for django https://github.com/mxschmitt/python-django-playwright, but it's dated.. so for now we'll put it on the to-do list.

An alternative that I had used in the past is Selenium: https://www.rockandnull.com/django-selenium/.

So maybe we open an issue with a todo on this topic.